src/main/java/com/savvato/tribeapp/controllers/PermissionsAPIController.java

```diff
@@ -3,24 +3,31 @@
 import com.savvato.tribeapp.controllers.dto.PermissionsRequest;
 import com.savvato.tribeapp.entities.User;
 import com.savvato.tribeapp.services.UserRoleMapService;
+import com.savvato.tribeapp.services.UserRoleService;
 import com.savvato.tribeapp.services.UserService;
+import com.savvato.tribeapp.entities.UserRole;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
 
 import javax.validation.Valid;
 
 @RestController
-@RequestMapping("/api/permissions")
 public class PermissionsAPIController {
     @Autowired
     UserRoleMapService userRoleMapService;
 
+    @Autowired
+    UserRoleService userRoleService;
+
     @Autowired
     UserService userService;
 
-    @GetMapping("/users")
+    @RequestMapping(value = { "/api/permissions/users"}, method=RequestMethod.GET)
```

Comment:
The endpoint mapping before is the correct way of doing it (GetMapping). Just FYI, we made a system wide change to that way of doing things a couple weeks ago.

Reply:
Done